### ISSUE TITLE
优化移动端的使用体验

### ISSUE THREAD

**Issue by Passerby1011** [ChatGPTBox-dev/chatGPTBox#408]
**Is your feature request related to a problem? Please describe.**
**优化移动端的使用体验，在移动端无法按快捷键，即便是可以安装扩展，扩展的选项也藏的很深，所以我希望移动端的扩展在使用时可以以侧边吸附悬浮球的形式存在，点击悬浮球进行弹窗和功能选择。是否启用移动模式可以手动设置

**Describe the solution you'd like**
我希望移动端的扩展在使用时可以以侧边吸附悬浮球的形式存在，点击悬浮球进行弹窗和功能选择

**Additional context**
**其他**

**Comment by josStorer** [comment]
可以，要晚一点，目前移动端主要用途还是搜索引擎集成和独立聊天界面

**Comment by calilex** [comment]
@josStorer Hi, is this feature still part of the plan? I assume you can add a QR code for tips in readme. Thanks for you awesome extension!

**Comment by josStorer** [comment]
@calilex Yes, it still in the plan. Due to the busyness of my work in real life and the need to take care of my family, I haven't had much time developing this feature recently, so I'm afraid it won't be implemented soon

**Comment by calilex** [comment]
I totally understand, thanks all the same. @josStorer
